contracts/0.8.9/oracle/AccountingOracle.sol

```diff
@@ -0,0 +1,817 @@
+        );
+
+        uint256 slotsElapsed = data.refSlot - prevRefSlot;
+
+        (/* address elRewardsVault */,
+            /* address safetyNetsRegistry */,
+            address stakingRouter,
+            /* address treasury */,
+            address withdrawalQueue,
+            /* address withdrawalVault */
+        ) = LOCATOR.coreComponents();
+
+        _processStakingRouterExitedKeysByModule(
+            IStakingRouter(stakingRouter),
+            boundaries,
+            data.stakingModuleIdsWithNewlyExitedValidators,
+            data.numExitedValidatorsByStakingModule,
+            slotsElapsed
+        );
+
+        IWithdrawalQueue(withdrawalQueue).updateBunkerMode(
+            data.isBunkerMode,
+            GENESIS_TIME + prevRefSlot * SECONDS_PER_SLOT
+        );
+
+        ILido(LIDO).handleOracleReport(
+            GENESIS_TIME + data.refSlot * SECONDS_PER_SLOT,
+            slotsElapsed * SECONDS_PER_SLOT,
+            data.numValidators,
+            data.clBalanceGwei * 1e9,
+            data.withdrawalVaultBalance,
+            data.elRewardsVaultBalance,
+            data.lastWithdrawalRequestIdToFinalize,
+            data.finalizationShareRate
+        );
+
+        _storageExtraDataProcessingState().value = ExtraDataProcessingState({
+            refSlot: data.refSlot.toUint64(),
+            dataFormat: data.extraDataFormat.toUint16(),
+            dataHash: data.extraDataHash,
+            itemsCount: data.extraDataItemsCount.toUint16(),
+            itemsProcessed: 0,
+            lastProcessedItem: 0
+        });
+    }
+
+    function _processStakingRouterExitedKeysByModule(
+        IStakingRouter stakingRouter,
+        DataBoundaries memory boundaries,
+        uint256[] calldata stakingModuleIds,
+        uint256[] calldata numExitedValidatorsByStakingModule,
+        uint256 slotsElapsed
+    ) internal {
+        if (stakingModuleIds.length != numExitedValidatorsByStakingModule.length) {
+            revert InvalidExitedValidatorsData();
+        }
+
+        if (stakingModuleIds.length == 0) {
+            return;
+        }
+
+        for (uint256 i = 1; i < stakingModuleIds.length;) {
+            if (stakingModuleIds[i] <= stakingModuleIds[i - 1]) {
+                revert InvalidExitedValidatorsData();
+            }
+            unchecked { ++i; }
+        }
+
+        uint256 exitedValidators = 0;
+        for (uint256 i = 0; i < stakingModuleIds.length;) {
+            if (numExitedValidatorsByStakingModule[i] == 0) {
+                revert InvalidExitedValidatorsData();
+            } else {
+                exitedValidators += numExitedValidatorsByStakingModule[i];
+            }
+            unchecked { ++i; }
+        }
+
+        uint256 prevExitedValidators = stakingRouter.getExitedKeysCountAcrossAllModules();
+        if (exitedValidators < prevExitedValidators) {
+            revert NumExitedValidatorsCannotDecrease();
+        }
+
+        uint256 exitedValidatorsPerDay =
```

Comment:
Pls, check this param using this method: https://github.com/lidofinance/lido-dao/blob/fcbe3996e00e05370f76a895fdd7482ba8aa1cd4/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L341

Reply:
done